windows/runner/run_loop.h

```diff
@@ -0,0 +1,40 @@
+#ifndef RUNNER_RUN_LOOP_H_
```

Comment:
What's with all these windows related files?

We're not creating a windows application, it's polluting the codebase